### PR TITLE
Feature/idea details

### PR DESCRIPTION
**Pull request by alxx88**
Add new page for details. Add new controller and service. Make unit tests from service.

### CODE REVIEW

File: src/main/webapp/css/ideaDetails.css [hunk 1]
```diff
@@ -0,0 +1,198 @@
+#shadowBox
```
Comment (Hanna-Aliakseichykava) [inline src/main/webapp/css/ideaDetails.css:1]: Please, rewrite comments in English

File: src/main/webapp/js/services.js [hunk 2]
```diff
@@ -12,4 +12,20 @@ function Rate(ideasFactory) {
           }
       };
 };
+
+angular.module('app.services')
+.service('detailsService',[detailsService]);
+function detailsService() {
```
Comment (Hanna-Aliakseichykava) [inline src/main/webapp/js/services.js:18]: services should not have state

File: src/main/webapp/js/ideas/ideas.controller.js [hunk 3]
```diff
@@ -30,6 +30,16 @@
...
             });
         });
+
+        vm.details = function(idea){
+            $scope.ideaDetail = {
+                title:idea.title,
+                description:idea.description,
+                createdAt:idea.createdAt
+            };
+            detailsService.setData($scope.ideaDetail);
```
Comment (Hanna-Aliakseichykava) [inline src/main/webapp/js/ideas/ideas.controller.js:40]: In this case it does not make sense to use the service. Having variable data is enough.

**Comment by Hanna-Aliakseichykava** [comment]
:+1:  (refactoring will be required later)

**Comment by reshetnev** [comment]
:+1:

**Comment by LostFan** [comment]
:+1:

**Pull request merged**